sound/soc/sof/intel/byt.c

```diff
@@ -455,6 +455,36 @@ static void byt_set_mach_params(const struct snd_soc_acpi_mach *mach,
 	mach_params->platform = dev_name(dev);
 }
 
+static void byt_reset_dsp_disable_int(struct snd_sof_dev *sdev)
+{
+	/* Disable Interrupt from both sides */
+	snd_sof_dsp_update_bits64(sdev, BYT_DSP_BAR, SHIM_IMRX, 0x3, 0x3);
+	snd_sof_dsp_update_bits64(sdev, BYT_DSP_BAR, SHIM_IMRD, 0x3, 0x3);
+
+	/* Put DSP into reset, set reset vector */
+	snd_sof_dsp_update_bits64(sdev, BYT_DSP_BAR, SHIM_CSR,
+				  SHIM_BYT_CSR_RST | SHIM_BYT_CSR_VECTOR_SEL,
+				  SHIM_BYT_CSR_RST | SHIM_BYT_CSR_VECTOR_SEL);
+}
+
+static int byt_suspend(struct snd_sof_dev *sdev, u32 target_state)
+{
+	sdev->dsp_power_state.state = target_state;
```

Comment:
@ranj063 Here ^(the original patch didn't have this)

Reply:
@eballetbo sorry I dont know how I missed this. we should remove it but it shouldnt make a difference anyway.

Reply:
You should remove that line but apply what @ranj063 proposed instead. 
* https://github.com/thesofproject/linux/pull/2094#issuecomment-628110819
Then works as before, suspend/resume works but suspend/resume while playing no. I can upload and updated patch

Reply:
Thanks, yes please add this patch so that we don' need to track review comments ;-)